bin/packages/get-babel-config.js

```diff
@@ -25,6 +25,8 @@ const babelConfigs = {
 		presets: map( babelDefaultConfig.presets, ( preset ) =>
 			overrideOptions( preset, '@babel/preset-env', {
 				modules: 'commonjs',
+				corejs: '3',
+				useBuiltIns: 'usage',
```

Comment:
This option will add polyfills such as Array.flat based on their usage in the code